port/common/omrmem32struct.h

```diff
@@ -41,6 +41,10 @@ typedef struct J9SubAllocateHeapMem32 {
 	J9HeapWrapper *subCommitHeapWrapper;
 	uintptr_t suballocator_initialSize;
 	uintptr_t suballocator_commitSize;
+	uintptr_t suballocator_incrementSize;
+#if defined(OMR_ENV_DATA64) && defined(LINUX)
+	uintptr_t suballocator_quickAlloc;
```

Comment:
The type should be `BOOLEAN` to match usage.

Reply:
Removing the comment isn't the right answer either.

Reply:
It feels a bit odd doing `... = (0 != value) ? 1 : 0` over just `... = (0 != value)` or just `... value` in the snippet that you provided. Do you have a strong preference here @keithc-ca?

Reply:
Sorry, that suggestion was only half-baked - it should have said
```c
		PPG_mem_mem32_subAllocHeapMem32.suballocator_quickAlloc = (0 != value) ? TRUE : FALSE;
```
For completeness, the reverse mapping in `omrport_copy_suballocator_heap_globals()` would be
```c
	omrport_control(destPortLibrary, OMRPORT_CTLDATA_ALLOCATE32_QUICK_ALLOC, subAllocGlobals.suballocator_quickAlloc ? 1 : 0);
```

Reply:
Ok. This is better. Thanks.